### PR TITLE
BLO-887 fix: add tab

### PR DESCRIPTION
**Pull request by simonheys**
### Changes

- add tab so it can receive messages

### CODE REVIEW

File: packages/extension/src/background/activeTabs.ts [hunk 1]
```diff
@@ -47,17 +47,28 @@ export async function sendMessageToHost(
 ): Promise<void> {
   const tabIds = await getTabIdsOfHost(host)
   const tabs = await activeTabs.get((tab) => tabIds.includes(tab.id))
-  await Promise.allSettled(tabs.map((tab) => tab.port.postMessage(message)))
+
+  for (const tab of tabs) {
+    try {
+      tab.port.postMessage(message)
```
Comment (janek26) [inline packages/extension/src/background/activeTabs.ts:53]: no await?

File: packages/extension/src/background/activeTabs.ts [hunk 2]
```diff
@@ -47,17 +47,28 @@ export async function sendMessageToHost(
...
   message: MessageType,
 ): Promise<void> {
   const tabs = await activeTabs.get()
 
-  const promises = tabs.map((tab) => tab.port.postMessage(message))
-
-  await Promise.allSettled(promises)
+  for (const tab of tabs) {
+    try {
+      tab.port.postMessage(message)
```
Comment (janek26) [inline packages/extension/src/background/activeTabs.ts:67]: same, does it catch errors from within the promise without await?

**Review by janek26 — APPROVED** [review]

File: packages/extension/src/background/activeTabs.ts (hunk 1 above)
Comment (simonheys) [reply packages/extension/src/background/activeTabs.ts:53]: Surprisingly turns out `port.postMessage` is not a Promise, so it was throwing exception in previous implementation. Actually should handle the error here and disconnect the port - it's usually an 'attempt to use disconnected port object' error

File: packages/extension/src/background/activeTabs.ts (hunk 2 above)
Comment (simonheys) [reply packages/extension/src/background/activeTabs.ts:67]: Same

**Review by dhruvkelawala — APPROVED** [review]

**Pull request merged**